bundles/at.medevit.elexis.agenda.ui/rsc/html/defaultParallel.html

```diff
@@ -155,7 +155,12 @@
 			if (event.description) {
 				var firstpart = event.description.split('<br /><br />');
 				if (firstpart.length > 0) {
-					element.find(".fc-title").append("<br/>" + firstpart[0]);
+					if (!event.title.trim()) {
+						element.find(".fc-title").text(firstpart[0]);
+					}
+					else {
+						element.find(".fc-title").append("<br/>" + firstpart[0]);
+					}
 				} else {
 					element.find(".fc-title").append("<br/>" + event.description);
 				}

```

Comment:
der grund ist in der description enthalten und wenn kein kontakt ausgewählt wurde ist dieser laut java code BLANK (Event.java // fullcalendar will no create title html div if no title is blank, add space) , somit sollte der grund immer angezeigt werden, wenn kein titel ermittelt werden kann.